cfn-resources/project/cmd/resource/resource.go

```diff
@@ -26,44 +26,41 @@ import (
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
-	"go.mongodb.org/atlas-sdk/v20231115002/admin"
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
 )
 
 var CreateRequiredFields = []string{constants.OrgID, constants.Name}
+var ReadRequiredFields = []string{constants.ID}
```

Comment:
knit: have only one var ReadUpdateDeleteRequiredFields

Reply:
Solved in 1c80bca